src/Compute/Compute/Models/PSVmssVMDiskEncryptionStatusContext.cs

```diff
@@ -23,13 +23,11 @@ class PSVmssVMDiskEncryptionStatusContext
         public List<DiskInstanceView> Disks { get; set; }
         public VirtualMachineExtensionInstanceView Extension { get; set; }
         public EncryptionStatus OsVolumeEncrypted { get; set; }
-        public EncryptionStatus DataVolumesEncrypted { get; set; }
+        public string DataVolumesEncrypted { get; set; }
```

Comment:
this may be considered a breaking change since the type is changing (even though it was returning incorrect responses and thus broken before) and as such might have special release requirements